### ISSUE TITLE
List: Screen readers are unable to announce the single selection case

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents#7905]
### Accessibility Issue with Screen Readers:

Screen readers are unable to announce the single selection case. In contrast, for multi-selection, screen readers successfully convey the MultiSelection message to sightless users.

### Isolated Example

https://m4kpxp.csb.app/

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

Both single selection or multiple selection should be recognized by screen readers.

### Screenshots or Videos

<img width="547" alt="image" src="https://github.com/SAP/ui5-webcomponents-react/assets/5724592/96c1a20c-16e2-430c-aa20-5eec25042896">

### UI5 Web Components for React Version

~1.22.0

### UI5 Web Components Version

~1.19.0

### Browser

Chrome

### Additional Context

I am from SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by PetyaMarkovaBogdanova** [comment]
Hi team,
can an accessibility expert, kindly, observe the text that has been read on selection Contains selectable items and Multi selectable items and decide if the first text needs to be enhanced to "Contains Single selectable items". Give in mind, that the read-out text also contains string "Single selection list" and "Multi selection list".

Best regards,
Petya Markova.
(UI5 WC Developer)

**Comment by dobrinyonkov** [comment]
Hello @sleric1024,

Thank you for your issue.

Actually, the case you have reported is currently a known issue and has been reported to the JAWS repository. For a more detailed explanation and discussion on this issue, you can visit the following link: https://github.com/FreedomScientific/standards-support/issues/440

Please do not hesitate to share any other issues you might encounter.

Kind Regards,
Dobrin
